package/com.unity.formats.usd/Tests/Editor/EditorAnalyticsBaseFixture.cs

```diff
@@ -0,0 +1,149 @@
+# if UNITY_2023_2_OR_NEWER
```

Comment:
Are we restricted to >= 2023.2 due to Vicky's Analytics fix on trunk? Or is there some other reason? Should we consider adding a note here about why it is 2023.2 or newer?

Reply:
yes the restriction is due to her fix only being on 2023.2+